doc/sphinx/language/gallina-specification-language.rst

```diff
@@ -1393,11 +1393,11 @@ Chapter :ref:`Tactics`. The basic assertion command is:
       The name you provided is already defined. You have then to choose
       another name.
 
-   .. exn:: Nested proofs are not allowed unless you turn option Nested Proofs Allowed on.
+   .. exn:: Nested proofs are not allowed unless you turn flag :flag:`Nested Proofs Allowed` on.
```

Comment:
You should not change this line unless you fix the error message directly in the code.